src/scripts/dialogConfig.js

```diff
@@ -128,8 +139,13 @@ const dialogConfig = (editor) => ({
       dialog.setValueOf('tab-basic', 'no-output', true);
     }
 
-    const codeArea = dialog.getContentElement('tab-basic', 'code').getElement();
-    codeArea.setHtml(editScriptAreaHTML(languageDictionary[language], code, output));
+    const language = getLanguage(editor);
```

Comment:
Just to check for understanding, is there no longer the need for the extra check between the dialog language and editor language? I vaguely remember that the dialog language was incorrect before but am having a little trouble understanding how this fixes it.

Reply:
Because dialog language is always set to default, which is python 3, when it shows. So checking the difference makes no sense. That why I introduced a global to store the `kernel` language instead of the previous dialog language. Let's make dialog language only refer to what the dropdown is selecting.

Reply:
That makes sense. Thank you for explaining.